modules/microshift-updating-rpms-y.adoc

```diff
@@ -0,0 +1,98 @@
+= Applying minor-version updates with RPMs
+
+Updating a {product-title} minor version on a non-OSTree system is a simple matter of downloading then updating the RPMs. For example, use the following procedure to upgrade from 4.14 to 4.15.
+
+[IMPORTANT]
+====
+You can only update {product-title} from one version to the next in sequence. Jumping minor versions is not supported. For example, must update 4.14 to 4.15.
+====
+
+.Prerequisites
+* The system requirements for installing {product-title} have been met.
+* You have root user access to the host.
+* The version of {product-title} you have is compatible to upgrade to the version you are preparing to use.
+* You have verified that your host operating system is compatible with the version of {product-title} you are preparing to install.
+* You have completed a system backup.
+* You have stopped {product-title} and any associated workloads.
+
+[NOTE]
+====
+You cannot downgrade {product-title} with this process. Downgrades are not supported.
+====
+
+.Procedure
+
+. As a root user, enable the {product-title} repositories by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-{rhel-major}-$(uname -m)-rpms
+----
+
+. Install {product-title} by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift
+----
+
+. Download your installation pull secret from the https://console.redhat.com/openshift/install/pull-secret[Red Hat Hybrid Cloud Console] to a temporary folder, for example, `$HOME/openshift-pull-secret`. This pull secret allows you to authenticate with the container registries that serve the container images used by {product-title}.
+
+. To copy the pull secret to the `/etc/crio` folder of your {op-system} machine, run the following command:
++
+[source,terminal]
+----
+$ sudo cp $HOME/openshift-pull-secret /etc/crio/openshift-pull-secret
+----
+
+. Make the root user the owner of the `/etc/crio/openshift-pull-secret` file by running the following command:
++
+[source,terminal]
+----
+$ sudo chown root:root /etc/crio/openshift-pull-secret
+----
+
+. Make the `/etc/crio/openshift-pull-secret` file readable and writeable by the root user only by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod 600 /etc/crio/openshift-pull-secret
+----
+
+. Update the {product-title} RPMs by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf update microshift
+----
++
+.Example output
++
+[source,terminal]
+----
+some output here
+----
+
+. Restart {product-title} to apply updates by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl start microshift
+----
+
+.Verification
+. Optional. Check the Greenboot logs to verify system health by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart greenboot-healthcheck
```

Comment:
I think it's more of a "start/retrigger a greenboot healthcheck to verify system health".
Logs can be obtained by running `journalctl -u greenboot-healthcheck`

Reply:
I put both option in there, but we might only need the start greenboot. See how it reads to you.